packages/organizations/origin-organization-irec-api/src/connection/handlers/get-accounts.handler.ts

```diff
@@ -21,6 +21,12 @@ export class GetAccountsHandler implements ICommandHandler<GetAccountsCommand> {
     ) {}
 
     async execute({ owner }: GetAccountsCommand): Promise<AccountDTO[]> {
-        return this.irecService.getAccountInfo(owner);
+        try {
+            return await this.irecService.getAccountInfo(owner);
+        } catch (e) {
+            // Registrant account does not have permissions to get list of accounts
+            // so we return empty list
+            return [];
```

Comment:
Shouldn't we throw a 403 instead?

Reply:
And return it with the description message that we got from I-REC API.

Reply:
I think no, we should not. It is just a list of accounts. And since the user does not have accounts empty list is better solution

Reply:
Let's at least add some logger log that says why we return an empty list.

Reply:
Something like "user with ID requested accounts but it has none".

Reply:
Will the I-REC API ever throw any other error besides this one when we do `getAccountInfo`?

Reply:
Would be better to do in catch something like 
```js
if (e.status === 403) {
    logger.log("....");
    return [];
}

throw e;
``` 

This way we would make sure that other errors are thrown and we can identify them, such as connection to I-REC API being broken etc.

Reply:
Done. Had to update irec client library for this